app/src/main/java/com/example/game/fragments/CreateEventFragment.java

```diff
@@ -124,13 +186,35 @@ public void onPickPhoto(View view) {
     }
 
     @Override
-    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+    public void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
         if (data != null && requestCode == PICK_PHOTO_CODE) {
             Uri photoUri = data.getData();
-            // Load the image located at photoUri into selectedImage
             Bitmap image = ImageUtil.loadFromUri(getContext(), photoUri);
             photoFile = new File(ImageUtil.saveToInternalStorage(getContext(), image));
             ivPoster.setImageBitmap(image);
         }
+        if (requestCode == AUTOCOMPLETE_REQUEST_CODE) {
```

Comment:
does not seems that the logic for this requestCode make sense. How could it be equals to 2 values at the same time?
Could be better if you change to more explicit if else or case switch